src/store/storage-controller.ts

```diff
@@ -23,10 +26,7 @@ export class StorageController {
    * @throws {DwnError} with `DwnErrorCode.MessageStoreDataSizeMismatch`
    *                    if `dataSize` in `descriptor` given mismatches the actual data size
    */
-  public static async put(
-    messageStore: MessageStore,
-    dataStore: DataStore,
-    eventLog: EventLog,
+  public async put(
```

Comment:
should we rename this `putWithData` to better differentiate it from `putMessage`(/`putMessageWithoutData`)?